src/frontend/screens/Settings/index.tsx

```diff
@@ -127,12 +104,7 @@ function Settings() {
       <SettingsContext.Provider value={contextValues}>
         <div className={`Settings ${type}`}>
           <div role="list" className="settingsWrapper">
-            <NavLink
-              to={returnPath}
-              role="link"
-              className="backButton"
-              state={{ gameInfo: gameInfo }}
-            >
+            <NavLink to="/" role="link" className="backButton">
```

Comment:
Since we moved the settings for specific games into a modal, this was always `/` anyway, we don't need all the code to detect to go back to a game page because that doesn't happen anymore

Reply:
This should go to `/library` for now (until #3540 is merged). Navigating to `/` just navigates to `/library` with an extra re-render